taichi_glsl/scalar.py

```diff
@@ -61,13 +61,14 @@ def sign(x, edge=0):
 
     :note:
         `sign(x, edge)` is equivalent with `sign(x - edge)`.
+        Currently Taichi use -1 to represent True. Consequently we add isTrue here.
 
     :return:
         The return value is computed as `(x >= edge) - (x <= edge)`,
         with type promoted.
     '''
     ret = x + edge  # type promotion
-    ret = (x >= edge) - (x <= edge)
+    ret = Truth(x >= edge) - Truth(x <= edge)
```

Comment:
Truth → truth to match the style of other functions?

Reply:
sure